### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by Rhodine-orleans-lindsay**
## What
- Amend kube files and config files as part of resolving [IMB-34](https://collaboration.homeoffice.gov.uk/jira/browse/IMB-34)
- allow data folder to be pushed to GitHub to allow uans to be retrieved
- Add warning log regarding data folder to server.js 
- Update yarn.lock

## Why
- To clean up kube files and resolve issues discovered such as :
  - large number of pods running in branch namespace which takes up a lot of resources
  - pods not tearing down over the weekend
  - uploaded case sheet in s3 bucket not being found when running app
  - updating rds image to most recent
  - updating readme documentation to include 
  - unblocking uan verification in branch and uat environment
 
**NB** - The warning log has been added to server.js as the data folder should not be deployed to production. A ticket will be created to work on resolving the bug that causes errors when the data folder is not included.

## How
- amended minimum and maximum number of pods for branch environment in hpa-ima.yml
- amended rds api image to most recent in rds deployment yml
- amended tear down branch url in deploy.sh
- amended readme
- amended config.js
- updated yarn.lock
- amend test setup to pass server unit test
- added warning log

## Testing
- Tests passing locally

**NB** - Snyk vulnerability needs to be updated in hof - a ticket is to be created for this

### CODE REVIEW

**Review by sulthan-ahmed — COMMENTED** [review]
Got says the excel file is empty is that right?

**Comment by Rhodine-orleans-lindsay** [comment]
> Got says the excel file is empty is that right?

This is file is created automatically when the server is started  and gets the object from the s3 bucket.  I have added the extension to .gitignore and it have tested in branch.

File: kube/autoscale/hpa-ima.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ spec:
   maxReplicas: 5
   {{ else }}
   minReplicas: 1
-  maxReplicas: 5
```
Comment (JHoldergov) [inline kube/autoscale/hpa-ima.yml:22]: are we sure we don't want to scale up to 5 incase of a high demand? 2 would usually be the minimum

**Review by JHoldergov — COMMENTED** [review]
see comments

File: kube/autoscale/hpa-ima.yml (hunk 1 above)
Comment (sulthan-ahmed) [reply kube/autoscale/hpa-ima.yml:22]: we usually do 3 @JHoldergov

**Comment by sulthan-ahmed** [comment]
is it worth explaining that @Rhodine-orleans-lindsay ?

File: kube/autoscale/hpa-ima.yml (hunk 1 above)
Comment (JHoldergov) [reply kube/autoscale/hpa-ima.yml:22]: right, so @Rhodine-orleans-lindsay can scale that to 3 then?

**Comment by sulthan-ahmed** [comment]
Great PR & commit message, would it be worth being more specific with the subject 
`e.g. Fix UAN DB issues & too many pods running`

File: kube/autoscale/hpa-ima.yml (hunk 1 above)
Comment (sulthan-ahmed) [reply kube/autoscale/hpa-ima.yml:22]: branch takes up a lot of resources, can we not have one? in prod yeah have 3

**Comment by Rhodine-orleans-lindsay** [comment]
> is it worth explaining that @Rhodine-orleans-lindsay ?

👍🏾 I have added comments in the file now @sulthan-ahmed

File: kube/autoscale/hpa-ima.yml (hunk 1 above)
Comment (JHoldergov) [reply kube/autoscale/hpa-ima.yml:22]: usually in a file we have kube/non-prod/hpa.yml and then kube/non-prod/hpa.yml if we are sure we have split the 2 then that is good but if not we may have to separate the two

Comment (JHoldergov) [reply kube/autoscale/hpa-ima.yml:22]: usually in a file we have kube/non-prod/hpa.yml and then kube/non-prod/hpa.yml if we are sure we have split the 2 then that is good but if not we may have to separate the two

Comment (JHoldergov) [reply kube/autoscale/hpa-ima.yml:22]: @sulthan-ahmed @Rhodine-orleans-lindsay ^^

Comment (Rhodine-orleans-lindsay) [reply kube/autoscale/hpa-ima.yml:22]: @JHoldergov there are if/else statements in the file e.g.` {{ if or (eq .KUBE_NAMESPACE .STG_ENV) (eq .KUBE_NAMESPACE .PROD_ENV) }`} scales the pods for the staging and prod environments

**Review by JHoldergov — APPROVED** [review]
I think we are good here

**Review by sulthan-ahmed — APPROVED** [review]
Great work @Rhodine-orleans-lindsay 

I won't hold this up anymore as it is blocking other people's work.  I would have liked more of an explanation about the UAN sheets and how to update it but shall we pick that up separately

**Pull request merged**